docs/getting_access/non_standard.md

```diff
@@ -0,0 +1,235 @@
+# Non-standard ways to use EESSI
+
+In case that an EESSI installation is not possible or desirable, there are a couple of "non-standard" ways to access EESSI.
+
+## Using cvmfsexec
+
+This option only works for RHEL-like systems and relies on the [cvmfsexec package](https://github.com/cvmfs/cvmfsexec) provided by CernVM-FS team. It will allow you to mount cvmfs as an unprivileged user, so no need to have a native cvmfs installation.
+
+### `cvmfsexec_eessi.sh`  wrapper script
+
+
+This wrapper script, `cvmfsexec_eessi.sh`, can be used to run a command in a subshell in which the EESSI CernVM-FS repository (`software.eessi.io`) is mounted via cvmfsexec`.
+
+```bash
+#!/bin/bash
+if [ -d /cvmfs/software.eessi.io ]; then
+    # run command directly, EESSI CernVM-FS repository is already mounted
+    "$@"
+else
+    # run command via in subshell where EESSI CernVM-FS repository is mounted,
+    # via cvmfsexec which is set up in a unique temporary directory
+    orig_workdir=$(pwd)
+    mkdir -p /tmp/$USER
+    tmpdir=$(mktemp -p /tmp/$USER -d)
+    cd $tmpdir
+    git clone https://github.com/cvmfs/cvmfsexec.git > $tmpdir/git_clone.out 2>&1
+    cd cvmfsexec
+    ./makedist default > $tmpdir/cvmfsexec_makedist.out 2>&1
+    cd $orig_workdir
+    $tmpdir/cvmfsexec/cvmfsexec software.eessi.io -- "$@"
+    # cleanup
+    rm -rf $tmpdir
+fi
+
+```
+
+Do make sure that this script is executable:
+
+```bash
+    chmod u+x ~/bin/cvmfsexec_eessi.sh
+```
+
+A simple way to test this script is to use it to inspect the contents of the EESSI repository:
+
+```bash
+    ~/bin/cvmfsexec_eessi.sh ls /cvmfs/software.eessi.io
+```
+
+or to start an interactive shell in which the EESSI repository is mounted:
+
+```bash
+    # Expected output from a machine without a cvmfs installation
+    $ ls /cvmfs
+    ls: cannot access '/cvmfs': No such file or directory
+
+    # Starting interactive shell
+    $ ~/bin/cvmfsexec_eessi.sh /bin/bash -l
+    CernVM-FS: loading Fuse module... done
+    CernVM-FS: mounted cvmfs on /tmp/hvela/tmp.iH6e993Adw/cvmfsexec/dist/cvmfs/cvmfs-config.cern.ch
+    CernVM-FS: loading Fuse module... done
+    CernVM-FS: mounted cvmfs on /tmp/hvela/tmp.iH6e993Adw/cvmfsexec/dist/cvmfs/software.eessi.io
+
+    # Now it's mounted!
+    $ ls /cvmfs/
+    cvmfs-config.cern.ch  software.eessi.io
+
+```
+
+Notice how now that EESSI is mounted, you will need to [set up the environment](https://www.eessi.io/docs/using_eessi/setting_up_environment/) in order to access the software itself.
+
+### `orted`  wrapper script
+
+In order to get multi-node runs of software working without having EESSI available system-wide, we also had to create a small wrapper script for the `orted` command that is used by Open MPI to start processes on remote nodes. This is necessary because `mpirun` launches `orted`, which must be run in an environment in which the EESSI repository is mounted. If not, MPI startup will fail with an error like
```

Comment:
```suggestion
In order to get multi-node runs of software working without having EESSI available system-wide, you also need to create a small wrapper script for the `orted` command that is used by Open MPI to start processes on remote nodes. This is necessary because `mpirun` launches `orted`, which must be run in an environment in which the EESSI repository is mounted. If not, MPI startup may fail with an error like
```